test/Distributed/distributed_actor_accessors.swift

```diff
@@ -0,0 +1,381 @@
+
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend-emit-module -emit-module-path %t/FakeDistributedActorSystems.swiftmodule -module-name FakeDistributedActorSystems -disable-availability-checking %S/Inputs/FakeDistributedActorSystems.swift
+// RUN: %target-swift-frontend -emit-irgen -enable-experimental-distributed -disable-availability-checking -I %t 2>&1 %s | %IRGenFileCheck %s
+
+// UNSUPPORTED: back_deploy_concurrency
+// REQUIRES: concurrency
+// REQUIRES: distributed
+
+import _Distributed
+import FakeDistributedActorSystems
+
+@available(SwiftStdlib 5.5, *)
+typealias DefaultDistributedActorSystem = FakeActorSystem
+
+enum SimpleE : Codable {
+case a
+}
+
+enum E : Codable {
+case a, b, c
+}
+
+enum IndirectE : Codable {
+  case empty
+  indirect case test(_: Int)
+}
+
+final class Obj : Codable, Sendable {
+  let x: Int
+
+  init(x: Int) {
+    self.x = x
+  }
+}
+
+struct LargeStruct : Codable {
+  var a: Int
+  var b: Int
+  var c: String
+  var d: Double
+}
+
+@available(SwiftStdlib 5.6, *)
+public distributed actor MyActor {
+  distributed func simple1(_: Int) {
+  }
+
+  // `String` would be a direct result as a struct type
+  distributed func simple2(_: Int) -> String {
+    return ""
+  }
+
+  // `String` is an object that gets exploded into two parameters
+  distributed func simple3(_: String) -> Int {
+    return 42
+  }
+
+  // Enum with a single case are special because they have an empty
+  // native schema so they are dropped from parameters/result.
+  distributed func single_case_enum(_ e: SimpleE) -> SimpleE {
+    return e
+  }
+
+  distributed func with_indirect_enums(_: IndirectE, _: Int) -> IndirectE {
+    return .empty
+  }
+
+  // Combination of multiple arguments, reference type and indirect result
+  //
+  // Note: Tuple types cannot be used here is either position because they
+  // cannot conform to protocols.
+  distributed func complex(_: [Int], _: Obj, _: String?, _: LargeStruct) -> LargeStruct {
+    fatalError()
+  }
+}
+
+@available(SwiftStdlib 5.6, *)
+public distributed actor MyOtherActor {
+  distributed func empty() {
+  }
+}
+
+
+/// ---> Let's check that distributed accessors and thunks are emitted as accessible functions
+
+/// -> `MyActor.simple1`
+// CHECK:      @"$s27distributed_actor_accessors7MyActorC7simple1yySiFTEHF" = private constant
+// CHECK-SAME: @"symbolic Si___________pIetMHygzo_ 27distributed_actor_accessors7MyActorC s5ErrorP"
```

Comment:
I just realized... `___` can be used to just ignore bits of text!?

Reply:
Ok not really, that's the actual text. :)

Reply:
It's the actual text, sorry, FileCheck cannot do that expect maybe by pattern matching